docs/index.rst

```diff
@@ -0,0 +1,39 @@
+##########################
+edx-platform Documentation
+##########################
+
+Developer documentation for `edx-platform` can be found in the following
+locations.
+
+* The `edx-platform docs directory`_ contains some local developer
+  documentation.
+
+* The `Developer Documentation Index`_ in Confluence provides additional links
+  to developer documentation for this and other projects. The rest of the `Open
+  edX Development space`_ in Confluence provides additional documentation.
+
+* User documentation and a more general Developer's Guide can be read on `Open
+  edX ReadTheDocs`_.  The source for these guides can be found in the
+  `edx-documentation`_ repository.
+
+.. _edx-platform docs directory: https://github.com/edx/edx-platform/tree/master/docs
+.. _Developer Documentation Index: https://openedx.atlassian.net/wiki/display/OpenDev/Developer+Documentation
+.. _Open edX Development space: https://openedx.atlassian.net/wiki/display/OpenDev/Open+edX+Development
+.. _Open edX ReadTheDocs: http://docs.edx.org/
+
+.. toctree::
+   :maxdepth: 2
+
+
+Change History
+**************
+
+* May, 2017: The local docs directory was cleared out to start fresh.
```

Comment:
@catong I just added a minor note here about the change.  I didn't want to leave all the details because it is available in git.  We could add: "including the removal of the "Open edX Platform API" guide what was on docs.edx.org."  

I just don't want to minimize text that isn't necessary.